spec/models/ability_spec.rb

```diff
@@ -25,6 +25,107 @@
         it { expect(ability).not_to be_able_to(:show, other_user) }
       end
 
+      describe 'who is allowed to see email address in user profile' do
+
+        # email address is hidden: admin, user's supervisor in current season (confirmed)
+        # email address is not hidden: admin, confirmed user, user him/herself
+
+        let(:other_user) { FactoryGirl.create(:user) }
+
+        context 'when email address is hidden' do
+          context 'when an admin' do
```

Comment:
Then I kinda need to add it to every context "when a [roleX], but not a [roleY], [conditionZ]?
e.g. "when a confirmed user, but not an admin", "when user's supervisor in current season (confirmed), but not an admin", "when an admin, but may be unconfirmed" etc.